docs/elements.md

```diff
@@ -675,7 +675,8 @@ Attribute | Type | Default | Description
 --- | --- | --- | ---
 `answers-name` | string | — | Variable name to store data in.
 `weight` | integer | 1 | Weight to use when computing a weighted average score over elements.
-`fixed-order` | boolean | False | Whether or not to display the statements in a fixed order; otherwise they are shuffled. Options are always shuffled.
+`fixed-order` | boolean | False | Whether or not to display the statements in a fixed order; otherwise they are shuffled.
+`fixed-options-order` | boolean | False | Whether or not to display the options in a fixed order; otherwise they are shuffled.
```

Comment:
It may be a good idea to include (either here or around line 697) in which order the options are printed if there are implicit options created based on statement matches (maybe encourage instructors to use explicit `pl-option` entries for all options to ensure the order is as expected).